exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusType.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.prometheus;
+
+import io.opentelemetry.sdk.metrics.data.AggregationTemporality;
+import io.opentelemetry.sdk.metrics.data.DoubleSumData;
+import io.opentelemetry.sdk.metrics.data.LongSumData;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+
+// Four types we use are same in prometheus and openmetrics format
+enum PrometheusType {
+  GAUGE("gauge"),
+  COUNTER("counter"),
+  SUMMARY("summary"),
+  HISTOGRAM("histogram");
+
+  private final String typeString;
+
+  PrometheusType(String typeString) {
+    this.typeString = typeString;
+  }
+
+  static PrometheusType forMetric(MetricData metric) {
+    switch (metric.getType()) {
+      case LONG_GAUGE:
+      case DOUBLE_GAUGE:
+        return GAUGE;
+      case LONG_SUM:
+        LongSumData longSumData = metric.getLongSumData();
+        if (longSumData.isMonotonic()
+            && longSumData.getAggregationTemporality() == AggregationTemporality.CUMULATIVE) {
+          return COUNTER;
+        }
+        return GAUGE;
```

Comment:
I believe after we added preferred aggregation support it isn't possible when using our SDK. It could be possible if someone used the exporter without our SDK though (creating arbitrary MetricData).